helpers.go

```diff
@@ -200,10 +201,10 @@ func preventImageOverwrite(repo, tag string) error {
 	imageExists, err := checkImageExists(repo, tag)
 
 	if err != nil {
-		return fmt.Errorf("Cannot proceed safely: %v.", err)
+		return fmt.Errorf("cannot proceed safely: %v", err)
 	}
 	if imageExists {
-		return fmt.Errorf("Cannot overwrite an existing image. Please use a different repository/tag.")
+		return errors.New("cannot overwrite an existing image. Please use a different repository/tag")
```

Comment:
this is just a nit. fmt.Errorf is used for formatting, there's no formatting done in this error, hence errors.new.